src/components/HistoryTables/MsigProposals/ProposalRow.tsx

```diff
@@ -45,11 +44,11 @@ export default function ProposalHistoryRow(props: ProposalHistoryRowProps) {
         </Badge>
       </TD>
       <TD>
-        <AddressWOptionalLink
-          onClick={e => e.stopPropagation()}
-          address={proposal.approved[0].robust || proposal.approved[0].id}
-          addressHref={addressHref}
-          inspectingAddress={inspectingAddress}
+        <AddressLink
+          id={proposal.approved[0].robust ? '' : proposal.approved[0].id}
+          address={proposal.approved[0].robust}
```

Comment:
Shouldn't this be:
```
id={proposal.approved[0].robust ? proposal.approved[0].robust : ""}
```

Reply:
See conversation above, however never pass an `address` to the `id` prop